### PR TITLE
[FIX] web: handle field stabilize order for new records on last page

### PR DESCRIPTION
**Pull request by seb-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by seb-odoo** [comment]
Hello @aab-odoo can you please review this PR? Small improvement to the sequence compute to prevent giving the same value to multiple records when possible.

File: addons/web/static/tests/fields/relational_fields_tests.js [hunk 1]
```diff
@@ -9552,6 +9552,74 @@ QUnit.module('relational_fields', {
...
+            res_id: 1,
+            viewOptions: {
+                mode: 'edit',
+            },
+        });
+
+        // click add a new line
+        // check turtle_int for new is the current max of the page
+        $('.o_field_x2many_list_row_add a').click();
+        assert.strictEqual($('.o_input[name="turtle_int"]').val(), '10');
```
Comment (aab-odoo) [inline addons/web/static/tests/fields/relational_fields_tests.js]: i'd prefix the selector with .modal

File: addons/web/static/tests/fields/relational_fields_tests.js [hunk 2]
```diff
@@ -9552,6 +9552,74 @@ QUnit.module('relational_fields', {
...
+            res_id: 1,
+            viewOptions: {
+                mode: 'edit',
+            },
+        });
+
+        // click add a new line
+        // check turtle_int for new is the current max of the page +1
+        $('.o_field_x2many_list_row_add a').click();
+        assert.strictEqual($('.o_input[name="turtle_int"]').val(), '22');
```
Comment (aab-odoo) [inline addons/web/static/tests/fields/relational_fields_tests.js]: same here

**Comment by seb-odoo** [comment]
I added the `.modal` prefix

**Comment by aab-odoo** [comment]
robodoo r+

**Pull request closed without merging**